pyproject.toml

```diff
@@ -194,6 +195,7 @@ exclude = [
     'marimo/_tutorials/',
     'marimo/_smoke_tests/',
 ]
+warn_unused_ignores=false
```

Comment:
ahhhhh! this was the worst without this since you couldn't do a blanket ignore b/c in another test matrix it would fail. and other matrices would have different ignores..

Reply:
haha yea it was getting really cumbersome